apps/api/src/app/events/usecases/send-message/send-message-in-app.usecase.ts

```diff
@@ -181,6 +181,14 @@ export class SendMessageInApp extends SendMessageBase {
       message = await this.messageRepository.findById(oldMessage._id);
     }
 
+    await this.queueService.wsSocketQueue.add({
+      event: 'notification_received',
+      userId: command.subscriberId,
+      payload: {
+        message,
+      },
+    });
```

Comment:
Is this the only place where this needed to be added? Did you had a chance to test it with the iframe embed or the useSocket from react?

Reply:
Yes what I can see this is the only place we need to add it.

Reply:
And yes I have test it with useSocket 😄